Assets/CameraScripts/PauseGame.cs

```diff
@@ -9,7 +9,7 @@ public class PauseGame : MonoBehaviour {
 	}
 
 	void Update(){
-		if(Input.GetKeyDown(KeyCode.Tab)){
+		if(Input.GetKeyDown(KeyCode.Escape)){
```

Comment:
Check if there is some way or need to prevent the default action of the esc key, we might need to do something like javascripts `event.preventDefault();`